modules/ROOT/pages/about-designing-a-mule-application.adoc

```diff
@@ -4,10 +4,50 @@ include::_attributes.adoc[]
 endif::[]
 :keywords:
 
-In Flow Designer, you create Mule applications as projects having one or more flows. A project contains the flows for one Mule application. A Mule application can have one or more flows. A flow consists of a sequence of _cards_, with each card representing a connector, core Mule Runtime component, API, or module. Each card receives an input, carries out a specific task, and then sends the output of that task to the next card in the flow. The final card in a flow is usually one that sends a notification that the Mule application has completed its task or that sends data.
+:imagesdir: ../assets/images
 
-To get started with Flow Designer by building an example Mule application, see xref:salesforce-to-twilio.adoc[Create Your First Mule Application with Flow Designer].
+In Flow Designer, you create Mule applications as projects having one or more flows. A project contains the flows for one Mule application. A Mule application can have one or more flows. A flow consists of a sequence of _cards_, with each card representing a connector, core Mule Runtime component, API, or module. Each card receives an input, carries out a specific task, and then sends the output of that task to the next card in the flow. The final card in a flow is usually one that sends a notification that the Mule application has completed its task or that sends data.
 
+== Getting Started
+
+To learn about Flow Designer by building an example Mule application, you can try the tutorial xref:salesforce-to-twilio.adoc[Create Your First Mule Application with Flow Designer]. The tutorial guides you in creating a Mule app that places bulk orders for company swag automatically when opportunities are closed, and that notifies customers via SMS about their order status and ETA.
+
+To start creating Mule applications:
+
+* Try the easiest option, which is to import a template into Flow Designer from Anypoint Exchange. See xref:import-templat.adoc[Create a Mule Application Quickly from a Template].
+* Create a new Mule-app project in Design Center and use the wizard that automatically opens to start a project with a trigger card and a target card.
+* Create a new Mule-app project and click *Go straight to the canvas* in the wizard to start building a flow from a blank canvas.
+
+== Tour of the Flow Designer Interface
+
+.The five areas of the Flow Designer interface
+image::FD-annotated.png[]
+
+. The canvas
++
+Here you can create a flow. In the top-left corner are the names of the project and the flow. The cards in a flow appear in the middle, in a sequence from left to right. The first card in a flow is the trigger card, which represents the connector and operation that you want to use to trigger the flow.
```

Comment:
I think this can be made simpler by deleting: "Here you can create a flow.  In the top-left corner are the names of the project and the flow."

Reply:
I moved "In the top-left corner are the names of the project and the flow." so that it comes after the paragraph. So, the text now starts "Here you can create a flow. The cards in a flow appear...".